bin/update

```diff
@@ -20,6 +20,9 @@ chdir APP_ROOT do
   puts "\n== Updating database =="
   system! 'bin/rails db:migrate'
 
+  puts "\n== Loading materials =="
+  system! 'bin/rails csv:load'
```

Comment:
I don't think this script runs on heroku. Would need to add it to a release section in the procfile

Reply:
Added to procfile as well